deploy/sre-prometheus/100-machine-api.PrometheusRule.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-machine-api-node-mismatch
+    role: alert-rules
+  name: sre-machine-api-node-mismatch
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-machine-api-node-mismatch
+    rules:
+    - alert: NoNodeObjectForMachineWarning
+      expr: &promql machine_api_status == 0
```

Comment:
Note: `&promql` and `kcslink` (later on line 20) establish two YAML anchors that are used on lines 21 and 26 respectively to write only one time the PromQL for this rule and the KCS link for the remediation document. The goal is to DRY out this file